rosbag_snapshot/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package format="2">
   <name>rosbag_snapshot</name>
-  <version>1.0.3</version>
+  <version>1.1.0</version>
```

Comment:
I'd recommend updating the patch version instead of minor so that we have an easier time incorporating upstream updates.